.rubocop_todo.yml

```diff
@@ -134,6 +134,10 @@ RSpec/MessageSpies:
     - 'spec/custom_facts/util/resolution_spec.rb'
     - 'spec/facter/facts/aix/os/name_spec.rb'
     - 'spec/facter/facts/aix/os/release_spec.rb'
+    - 'spec/facter/facts/freebsd/os/architecture_spec.rb'
+    - 'spec/facter/facts/freebsd/os/name_spec.rb'
+    - 'spec/facter/facts/freebsd/ruby/platform_spec.rb'
+    - 'spec/facter/facts/freebsd/ruby/sitedir_spec.rb'
```

Comment:
These offences should be fixes.

e.g.
```ruby
    it 'calls Facter::Resolvers::Uname' do
      fact.call_the_resolver
      expect(Facter::Resolvers::Uname).to have_received(:resolve).with(:machine)
    end
```

Reply:
I amended the relevant commits accordingly, thanks!